doc/devel/release.md

```diff
@@ -0,0 +1,66 @@
+# kube-spawn release guide
+
+## Release cycle
+
+This section describes the typical release cycle of kube-spawn:
+
+1. A GitHub [milestone][milestones] sets the target date for a future kube-spawn release.
+2. Issues grouped into the next release milestone are worked on in order of priority.
+3. Changes are submitted for review in the form of a GitHub Pull Request (PR). Each PR undergoes review and must pass continuous integration (CI) tests before being accepted and merged into the main line of kube-spawn source code.
+4. The day before each release is a short code freeze during which no new code or dependencies may be merged. Instead, this period focuses on polishing the release, with tasks concerning:
+  * Documentation
+  * Usability tests
+  * Issues triaging
+  * Roadmap planning and scheduling the next release milestone
+  * Organizational and backlog review
+  * Build, distribution, and install testing by release manager
+
+## Release process
+
+This section shows how to perform a release of kube-spawn.
+Only parts of the procedure are automated; this is somewhat intentional (manual steps for sanity checking) but it can probably be further scripted, please help.
```

Comment:
Rather than *please help*, I would write something like *help is appreciated*

Reply:
updated.

(The phrasing comes from rkt)